recipes/fatiando/meta.yaml

```diff
@@ -0,0 +1,73 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "fatiando" %}
+{% set version = "0.4" %}
+{% set sha256 = "7709a4f7e40fd94bce8f0825d1d33094d271052374707bf2595a2cbd314401c7" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  #script: python setup.py install --single-version-externally-managed --record record.txt
+  script: python setup.py install
```

Comment:
It looks like fatiando does not support python 3 yet (http://www.fatiando.org/install.html) so you'll want to add `skip: True # [py3k]` in the `build` section.